adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/api/rest/v1/DataImportEndpoint.java

```diff
@@ -0,0 +1,64 @@
+package org.jvalue.ods.adapterservice.datasource.api.rest.v1;
+
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+import org.jvalue.ods.adapterservice.datasource.DatasourceManager;
+import org.jvalue.ods.adapterservice.datasource.model.DataImport;
+import org.jvalue.ods.adapterservice.datasource.model.exceptions.*;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(Mappings.DATASOURCE_PATH)
```

Comment:
```suggestion
@RequestMapping(Mappings.DATASOURCE_PATH + "/{datasourceId}" + Mappings.DATAIMPORT_PATH)
```
I think we can specify more of the common path prefix here to make the `@GetMapping(...)` shorter and remove duplication.